.github/workflows/deploy-experimental.yml

```diff
@@ -156,6 +156,7 @@ jobs:
           pushd solution/backend
           npm install serverless@">=3.38.0 <4" -g
           npm install
+          serverless deploy --config ./serverless-maintenance.yml --stage maintenance${PR} | tee output.log
```

Comment:
This will be removed when we go to prod since we wont be placing our experimental in maintenance mode.